README.md

```diff
@@ -104,23 +113,68 @@ $ docker-compose ps
 $ docker-compose logs -f
 ```
 
-**Note**: 
-There is a time drift issue on Docker for Mac, to solve it try running [Fixing Time drift issue on Docker for Mac](https://blog.shameerc.com/2017/03/quick-tip-fixing-time-drift-issue-on-docker-for-mac):
+## Running on DappNode
+
+Building and running the TrueBlocks docker image is taken care of by the dAppNode package manager. If you are running a dAppNode, you can simply install the package from the dAppNode package manager.
```

Comment:
We can leave this for later, but we can probably tell the user what the package name will be that they should look for. My guess is that there's a growing number of packages already up there.

Reply:
Good idea. Let's leave it for later.